projects/wvr-elements/src/lib/wvr-list/wvr-list-item/wvr-list-item.component.scss

```diff
@@ -0,0 +1,201 @@
+@import "../../shared/styles/host-styles.scss";
+
+:host {
+
+  ::ng-deep {
```

Comment:
Could this be moved below variable declarations

Reply:
Done